src/pages/Markets/hooks/useCoingeckoData.ts

```diff
@@ -138,16 +208,46 @@ export const useRecentlyAddedQuery = ({ enabled = true }: { enabled?: boolean })
 export const useMarketsQuery = ({
   enabled = true,
   orderBy,
+  sortBy,
 }: {
-  orderBy: 'market_cap_desc' | 'volume_desc'
+  orderBy?: OrderOptionsKeys
+  sortBy?: SortOptionsKeys
   enabled?: boolean
 }) => {
   const dispatch = useAppDispatch()
   const assets = useAppSelector(selectAssets)
 
+  const prefixOrderBy = (() => {
+    switch (sortBy) {
+      case SortOptionsKeys.MARKET_CAP:
+        return 'market_cap'
+      case SortOptionsKeys.VOLUME:
+        return 'volume'
+      default:
+        return 'volume'
+    }
+  })()
+
+  const sort = (() => {
+    switch (orderBy) {
+      case OrderOptionsKeys.ASCENDING:
+        return 'asc'
+      case OrderOptionsKeys.DESCENDING:
+        return 'desc'
+      default:
+        return 'desc'
+    }
+  })()
+
+  const order = `${prefixOrderBy}_${sort}` as
+    | 'market_cap_asc'
+    | 'market_cap_desc'
+    | 'volume_desc'
+    | 'volume_asc'
```

Comment:
```suggestion
  const order: CoinGeckoSortKey = `${prefixOrderBy}_${sort}`
```

Reply:
Oh nice! Changed in [1ac600a](https://github.com/shapeshift/web/pull/8006/commits/1ac600aeb95da5729ef78865c837fd0bcec0f777)